src/jit/lsra.h

```diff
@@ -30,10 +30,35 @@ const unsigned int   MaxLocation = UINT_MAX;
 const unsigned int MaxInternalRegisters = 8;
 const unsigned int RegisterTypeCount    = 2;
 
+/*****************************************************************************
+* Register types
+*****************************************************************************/
 typedef var_types RegisterType;
 #define IntRegisterType TYP_INT
 #define FloatRegisterType TYP_FLOAT
 
+template <class T>
+RegisterType regType(T type)
```

Comment:
Needs a method header comment

Reply:
Again, all these were pre-existing but I will add method headers.

Reply:
Note that these are pretty simplistic, so I will us abbreviated headers.